### ISSUE TITLE
The actual code

### ISSUE THREAD

**Issue by xiaoxiaoSummer** [pedrormjunior/ossvm-results#1]
Hi,

Can i ask when the actual code release thanks.

**Comment by pedrormjunior** [comment]
Hello Guoxia,

I hope we will be able to release the code soon. Our paper is pending approval for publication. I think it will not take too long though.

Bests.

**Comment by AnnaFiorini** [comment]
Hi,

Is there any news about the release?

Thanks

**Comment by pedrormjunior** [comment]
Hello Anna, I hope it is almost there. We've just submitted a revision of our paper. Some coauthors prefer not to release the code before publication, so we need a little more patience.

**Comment by pedrormjunior** [comment]
Hello colleagues, the code were pushed today in another repository [1]. Notice that the name of the repository were changed. Soon we will publish on arXiv the updated version of our paper. We had changed the name of the method from Specialized Support Vector Machines to Open-Set Support Vector Machines, hence the update on the repo name. If you want details beyond paper's content, visit the Supplementary Material [2].

I apologize for not being able to make it available before.

[1] See commit pedrormjunior/ossvm@50d51dc482c8e13df7d9037976b97db7e60a1ccf.
[2] https://pedrormjunior.github.io/OSSVM.html

PS: Let us know if you have any doubts on using it.  The only novel documentation by now is the mentioned commit [1].
